packages/framework/src/Twig/ImageExtension.php

```diff
@@ -244,4 +264,59 @@ protected function getImageHtmlByEntityName(array $attributes, $entityName, $add
             'imageCssClass' => $this->getImageCssClass($entityName, $attributes['type'], $attributes['size']),
         ]);
     }
+
+    /**
+     * @param array $attributes
+     * @return bool
+     */
+    protected function shouldUseLazyLoad(array $attributes): bool
+    {
+        // no-image should not be lazy loaded
```

Comment:
Do you think that this is not clear? If so move that condition into method with name similar to this comment

Reply:
Removed the comment, I believe it's clear enough. In 73e9ce93a